content/strategy-goals/strategy/growth-team/index.md

```diff
@@ -2,7 +2,7 @@
 
 ## Vision
 
-In a year, Sourcegraph will have a product-led, cloud-first growth motion. The signup flow will be a fast self-service experience that lets users pick the right deployment option for them. After signup, users will discover the value of Sourcegraph in a few minutes and have a guided path to becoming a power user, that will lead them to share Sourcegraph with their teammates and power viral growth within a given customer. Sourcegraph's experience will be oprimised for collaboration. This will allow Sourcegraph the company to have a product-led-growth motion and to deliver its vision to make it so everyone can code.
+In a year, Sourcegraph will have a product-led, cloud-first growth motion. The signup flow will be a fast self-service experience that lets users pick the right deployment option for them. After signup, users will discover the value of Sourcegraph in a few minutes and have a guided path to becoming a power user, that will lead them to share Sourcegraph with their teammates and power viral growth within a given customer. Sourcegraph's experience will be optimised for collaboration. This will allow Sourcegraph the company to have a product-led-growth motion and to deliver its vision to make it so everyone can code.
```

Comment:
Might be worth a mention of new business focus vs existing customer expansion.  Not sure if that is implied by growth.

Reply:
I don't think that is exactly the case over 1 year: given we are single-tenant, virality/feedback loop will primarily be within an account. This is required, though: we need new accounts to have more than the initial user.